h5c/vic/src/vic-webapp/src/app/summary-view/summary-view.component.ts

```diff
@@ -96,6 +97,7 @@ export class VicSummaryViewComponent implements OnInit, OnDestroy {
       this.createWzService.verifyVicMachineApiEndpoint()
       .subscribe(
         (ip: string) => {
+          this.applianceIp = ip;
```

Comment:
It seems like we only set the IP if an error does not occur. Given the goal of aiding a user in debugging, it seems like we would also want to set the IP in that case too.